setup.py

```diff
@@ -11,11 +11,25 @@
 README_FILE = PROJECT_ROOT / "README.md"
 VERSION_FILE = PROJECT_ROOT / "pymc4" / "__init__.py"
 
-NAME = "PyMC4"
+NAME = "pymc4"
```

Comment:
just curious what differences does this make? Does it install as "PyMC4" with pip or vnenv in the config?

Reply:
I think pypi uses this